README.md

```diff
@@ -1,3 +1,18 @@
 # mozilla-repo-urls
 
 Process Mozilla's repository URLs. The intent is to centralize URLs parsing.
+
+## Running tests
+
+1. `uv tool install tox --with tox-uv`
+1. `uv tool run tox -e py39`
```

Comment:
Nit: `py39` is not in `tox.ini` anymore. Let's bump the python version here given that we dropped the support of older versions of Python